locations/spiders/travelex.py

```diff
@@ -0,0 +1,42 @@
+import scrapy
+
+from locations.items import Feature
+
+
+class TravelexSpider(scrapy.Spider):
+    name = "travelex"
+    item_attributes = {"brand": "Tranvelex", "brand_wikidata": "Q2337964"}
+    allowed_domains = ["https://www.travelex.co.uk/"]
+
+    def start_requests(self):
+        yield scrapy.Request(
+            "https://api.travelex.net/salt/store/search?key=Travelex&mode=storeLocator&lat=52.0&lng=5.0",
+        )
+
+    def parse(self, response):
+        data = response.json()
+
+        item_categories = data.get("items")
+        for category in item_categories:
+            stores = category.get("stores")
+            print(len(stores))
+            for row in stores:
+                properties = {
+                    "ref": row.get("storeId"),
+                    "name": row.get("name"),
+                    "addr_full": row.get("formattedAddress"),
+                    "brand": self.item_attributes.get("brand"),
+                    "street_address": row.get("address").get("address1"),
+                    "city": row.get("address").get("city"),
+                    "postcode": row.get("address").get("postalCode"),
+                    "lat": float(row.get("lat")),
+                    "lon": float(row.get("lng")),
+                    "website": row.get("storeUrl"),
+                    "extras": {
+                        "directions": row.get("directions"),
+                        "notes": row.get("notes"),
+                        "terminal": row.get("terminal")
+                    }
+                }
+
+                yield Feature(**properties)
```

Comment:
Use DictParser here? Sample:


from locations.dict_parser import DictParser

item = DictParser.parse(row)
item["whatever"] = row["something"] # Adjust fields that DictParser misses or gets wrong
...
yield item